### PR TITLE
features: gatekeeper: update source package name

### PR DESCRIPTION
**Pull request by sabinaaledort**
The name of the gatekeeper-operator package in redhat-operators catalogsource was updated.

### CODE REVIEW

**Comment by sabinaaledort** [comment]
/hold
trying to understand why the name of the package was changed.

**Comment by sabinaaledort** [comment]
failing because the v0.2.3 gatekeeper is pointing to an u/s image that doesn't exist
 https://github.com/stolostron/gatekeeper-operator/blob/v0.2.3/bundle/manifests/gatekeeper-operator.clusterserviceversion.yaml#L321

File: hack/setup-build-index-image.sh [hunk 1]
```diff
@@ -90,8 +90,10 @@ spec:
           cd ..
 
 
-          git clone --single-branch --branch GATEKEEPER_VERSION https://github.com/open-cluster-management/gatekeeper-operator.git
+          git clone --single-branch --branch GATEKEEPER_VERSION https://github.com/stolostron/gatekeeper-operator.git
           cd gatekeeper-operator
+          # replace u/s image with d/s
+          sed -i 's/quay.io\/gatekeeper\/gatekeeper-operator/registry.redhat.io\/rhacm2\/gatekeeper-rhel8-operator/g' bundle/manifests/gatekeeper-operator.clusterserviceversion.yaml
```
Comment (SchSeba) [inline hack/setup-build-index-image.sh:96]: I don't think we should do something like that here

Comment (sabinaaledort) [reply hack/setup-build-index-image.sh:96]: I don't think we need this change anymore. I did that to fix ci errors, but the issue was resolved, and anyway we are checking if gatekeeper should still be tested, so maybe we shouldn't touch or change is now. I'm closing this PR.

**Comment by sabinaaledort** [comment]
Did that to fix ci errors, but the issue was resolved, and anyway we are checking if gatekeeper should still be tested, so maybe we shouldn't touch or change is now.

**Pull request closed without merging**